AzureData/Source/DocumentClient.swift

```diff
@@ -226,7 +226,12 @@ public class DocumentClient {
 
 
     // replace
-    // TODO: replace
+    public func replace (inCollectionWithId collectionId: String, fromDatabase databaseId: String, policy: DocumentCollection.IndexingPolicy, callback: @escaping (Response<DocumentCollection>) -> ()) {
+
+        let resourceLocation: ResourceLocation = .collection(databaseId: databaseId, id: collectionId)
+
+        return self.update(resourceWithId: collectionId, withData: policy.asJSONDictionary, at: resourceLocation, callback: callback)
```

Comment:
This should be `return self.update(resourceWithId: collectionId, withData: ["indexingPolicy": policy.asJSONDictionary], at: resourceLocation, callback: callback`. Will fix in a bit.

Reply:
instead of this, just init a new `DocumentCollection` and set the `indexingPolicy` property and call `replace`:

```swift
public func replace (collectionWithId collectionId: String, inDatabase databaseId: String, usingPolicy policy: DocumentCollection.IndexingPolicy, callback: @escaping (Response<DocumentCollection>) -> ()) {
    
    let resourceLocation: ResourceLocation = .collection(databaseId: databaseId, id: collectionId)

    var collection = DocumentCollection(collectionId)
    
    collection.indexingPolicy = policy
    
    return self.replace(collection, at: resourceLocation, callback: callback)    
}
```